app/config/question-bank.js

```diff
@@ -1696,12 +1696,17 @@ const questionBank = {
               title: 'Single Business Identifier (SBI)',
               classes: 'govuk-input govuk-input--width-10',
               label: {
-                text: 'Single Business Identifier (SBI)',
+                html: `
```

Comment:
this will create accessibility issue can you change like 
l**abel: {
                text: 'Single Business Identifier (SBI)',
                classes: 'govuk-label'
              },
              hint: {
                html: 'If you do not have an SBI, you will need to get one for full application'
              },**